source/2_platform/rex_windows/src/entrypoint.cpp

```diff
@@ -19,6 +19,11 @@
 INT APIENTRY WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPTSTR lpCmdLine, int nShowCmd)
 {
   rex::CommandLineArguments cmd_args(GetCommandLineA());
+
+#if REX_DEBUG
```

Comment:
we need to define which macros we use to specify certain configs because REX_DEBUG sounds like it won't be active in debug_opt